package.json

```diff
@@ -32,8 +32,8 @@
     "@foxglove/eslint-plugin": "file:.",
     "@foxglove/tsconfig": "1.0.0",
     "@types/jest": "^27",
-    "@typescript-eslint/eslint-plugin": "^4",
-    "@typescript-eslint/parser": "^4",
+    "@typescript-eslint/eslint-plugin": "^4.31.0",
+    "@typescript-eslint/parser": "^4.31.0",
```

Comment:
I was tempted to change these to peerDependencies, so that downstream users will receive appropriate warnings if an older version is installed. However, I recognize that some may use this plugin for pure JS code and don't need the TypeScript related plugins. @amacneil is there a more appropriate way to ensure the user will always import a new enough version?

Reply:
Yeah - same thing applies for typescript itself. I opted for just not including these. This situation with a new rule is unique though. 

The only option here would be to split out the packages so they can have separate deps, but that seems annoying/overkill. Otherwise we could list it as an optionalPeerDep which I think only yarn supports, and I’m not sure is even enforced anywhere. 

Given that we are the primary users of this package and we tend to keep our deps up to date, I’d be inclined to just leave things as is and put a PSA in slack (mostly for all @jhurliman’s packages).